README.md

```diff
@@ -243,7 +243,7 @@ _path/to/myapp/linkerd.yml_, you could start linkerd in docker with
 the following command:
 
 ```
-$ docker run -p 4140:4140 -v path/to/myapp:/myapp -w /myapp buoyantio/linkerd:0.0.10-SNAPSHOT linkerd.yml
+$ docker run -p 4140:4140 -v /absolute/path/to/myapp:/myapp -w /myapp buoyantio/linkerd:0.0.10-SNAPSHOT linkerd.yml
```

Comment:
Ah, hmm -- I think we're probably using different versions of docker.  Mine supports mounting files [[link](https://docs.docker.com/engine/userguide/containers/dockervolumes/#mount-a-host-file-as-a-data-volume)].  But no matter, if it's not universally supported, I agree that mounting a directory and using an absolute path to get rid of the -w flag is the right approach.  Thanks!